.github/workflows/kubernetes.yml

```diff
@@ -57,6 +57,17 @@ jobs:
           max_attempts: 10
           timeout_minutes: 45
           command: kind get clusters | xargs -I {} kind delete cluster --name {} && cargo test --locked -p linera-service --features scylladb,kubernetes --test end_to_end_tests -- kubernetes
+      - name: Port forward Prometheus
+        run: |
+          kubectl port-forward prometheus-linera-core-kube-prometheu-prometheus-0 9090
```

Comment:
We'll have 4 clusters on CI, but since it doesn't matter we can just port forward whatever the default one is